types/mathjs/mathjs-tests.ts

```diff
@@ -33,7 +33,7 @@ Bignumbers examples
 {
 	// configure the default type of numbers as BigNumbers
 	math.config({
-		number: 'bignumber',
+		number: 'BigNumber',
```

Comment:
Test gave a warning that 'bignumbers' should be 'BigNumbers' so I changed it

Reply:
Perfect.

Reply:
Actually wait, I'm eating my words on this a bit. Did the TS compiler complain, or did mathjs? Because we should make sure that `'bignumber'` fails to compile if it is wrong. (probably by changing the `ConfigOptions.number` to be type: `'BigNumber' | 'number' | ...`

Reply:
mathjs complained, but it was just a warning not a fatal error.  So i think 'bignumber' is allowed, just not preferred.